docker/production/app/Dockerfile

```diff
@@ -16,10 +16,6 @@ ENV RAILS_ENV='' \
     POSTGRES_USER='postgres' \
     POSTGRES_DB='' \
     SECRET_KEY_BASE='' \
-    EMAIL_PORT='' \
-    EMAIL_HOST='' \
-    EMAIL_FROM_ADDR='' \
-    EMAIL_DEFAULT_URL_HOST=''
```

Comment:
These ENV vars aren't used by the `production` docker config, nor are they used by the application.

Reply:
@grenewode FYI ☝️